src/components/Block/SpaceFormProfile.vue

```diff
@@ -14,36 +20,61 @@ const definition = {
   type: 'object',
   title: 'Space',
   additionalProperties: false,
-  required: ['name'],
+  required: ['name', 'treasuryAddress'],
   properties: {
     name: {
       type: 'string',
       title: 'Name',
       minLength: 1,
       examples: ['Space name']
     },
-    about: {
+    description: {
       type: 'string',
       format: 'long',
       title: 'About',
       examples: ['Space description']
     },
-    website: {
+    externalUrl: {
       type: 'string',
       title: 'Website',
-      examples: ['Space website URL']
+      examples: ['Website URL']
+    },
+    githubUrl: {
+      type: 'string',
+      title: 'Github',
```

Comment:
Should be written with the official casing
```suggestion
      title: 'GitHub',
```